src/templates/XmlRenderer/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { XMLRenderer } from "./template";
+
+export const XMLRendererTemplate = [
+  {
+    id: "XML_RENDERER",
+    label: "XML_Renderer",
```

Comment:
label is like a name nia, no need underscore

Reply:
Done.